zebrad/Cargo.toml

```
@@ -280,16 +280,16 @@ proptest-derive = "0.4.0"
 # enable span traces and track caller in tests
 color-eyre = { version = "0.6.2" }
 
-zebra-chain = { path = "../zebra-chain", features = ["proptest-impl"] }
-zebra-consensus = { path = "../zebra-consensus", features = ["proptest-impl"] }
-zebra-network = { path = "../zebra-network", features = ["proptest-impl"] }
-zebra-state = { path = "../zebra-state", features = ["proptest-impl"] }
-zebra-scan = { path = "../zebra-scan", features = ["proptest-impl"] }
+zebra-chain = { path = "../zebra-chain", version = "1.0.0-beta.34", features = ["proptest-impl"] }
+zebra-consensus = { path = "../zebra-consensus", version = "1.0.0-beta.34", features = ["proptest-impl"] }
+zebra-network = { path = "../zebra-network", version = "1.0.0-beta.34", features = ["proptest-impl"] }
+zebra-scan = { path = "../zebra-scan", version = "0.1.0-alpha.3", features = ["proptest-impl"] }
+zebra-state = { path = "../zebra-state", version = "1.0.0-beta.34", features = ["proptest-impl"] }
 
-zebra-node-services = { path = "../zebra-node-services", features = ["rpc-client"] }
+zebra-node-services = { path = "../zebra-node-services", version = "1.0.0-beta.34", features = ["rpc-client"] }
 
-zebra-test = { path = "../zebra-test" }
-zebra-grpc = { path = "../zebra-grpc" }
+zebra-test = { path = "../zebra-test" version = "1.0.0-beta.34" }
+zebra-grpc = { path = "../zebra-grpc" version = "0.1.0-alpha.1" }
```

missed commas, sorry.

```suggestion
zebra-test = { path = "../zebra-test", version = "1.0.0-beta.34" }
zebra-grpc = { path = "../zebra-grpc", version = "0.1.0-alpha.1" }
```